source/sustain/engage_with_stakeholders.html.erb.md

```diff
@@ -5,13 +5,13 @@ layout: activity
 ---
 
 {:.introduction}
-Continued engagement with stakeholders should seek to ensure that they are aware of the improvements in accessibility and resulting benefits. Internal stakeholders may be interested in how this is impacting on the business and the effects on key performance indicators.
+Continued engagement with stakeholders will ensure that they are aware of the improvements in accessibility and resulting benefits. Internal stakeholders may be interested in how this is impacting the business and the effects on key performance indicators.
 
 External stakeholders are more likely to be interested in how your organization is progressing towards fulfilling stated accessibility goals. Keep working with external suppliers to improve the services and tools that they deliver.
 
-Work with internal stakeholders to ensure that accessibility remains a priority, particularly when it comes to new projects. Long-term engagement should also consider how accessibility changes are impacting on project delivery. Are projects costing more? Is less time needed to ensure that accessibility is covered? Are there less problems at the end of projects?
+Work with internal stakeholders to ensure that accessibility remains a priority, particularly when it comes to new projects. Long-term engagement should also consider how accessibility changes are impacting project delivery. Are projects costing more? Is less time needed to ensure that accessibility is covered? Are there less problems at the end of projects?
 
-Changes in organizational structure may also be considered in the long-term and requires the strategic view that senior stakeholders have. For example, providing subject matter experts within each development team, building accessibility support services within the organization, or keeping accessibility a priority during restructuring.
+Changes in organizational structure may also be considered in the long-term and require the strategic view that senior stakeholders have. Examples include: providing subject matter experts within each development team, building accessibility support services within the organization, or keeping accessibility a priority during restructuring.
```

Comment:
I will merge this, but likely change the 'Examples include: ...' to be a proper list. I would prefer to avoid using ':' in the middle of the sentence... even correctly ;)